pkg/resource/scalable_target/custom_api.go

```diff
@@ -31,3 +34,9 @@ func (rm *resourceManager) customDescribeScalableTarget(
 		input.SetResourceIds(resourceIDList)
 	}
 }
+
+// customSetOutputUpdate sets the LastModifiedTime field to the current time post an update
+func (rm *resourceManager) customSetLastModifiedTime(ko *svcapitypes.ScalableTarget) {
+	currentTime := metav1.Time{Time: time.Now()}
```

Comment:
Here and below, please use [`time.Now().UTC()`](https://pkg.go.dev/time#Time.UTC)

Reply:
yes I noticed that later - thank you!

Reply:
done.